### ISSUE TITLE
Triggering single row rerender onHover in DetailsList

### ISSUE THREAD

**Issue by Dan86de** [microsoft/fluentui#16662]
Would be nice to have possibility to just rerender single row in DetailsList. Is there any way to do that? I have DetailsList with several components inside each of the rows. I would like for example triggering change in IconButton iconName prop when im hovering on single row. Until now i didn't find the way to do this properly.

![image](https://user-images.githubusercontent.com/23312063/106000032-aabcad80-60ae-11eb-8dad-5d93b6b190a9.png)

**Comment by gouttierre** [comment]
@Dan86de - Thanks for filing this, but you’ll need to add a CodePen repro before we can help. You can start from one of the COMPONENT examples (click “Export to CodePen”) or https://aka.ms/fluentpen.
